dsaps/models.py

```diff
@@ -186,19 +189,21 @@ class Item(BaseRecord):
     file_identifier = Field()
     source_system_identifier = Field()
 
-    def bitstreams_from_directory(self, directory, file_type='*'):
+    def get_bitstreams_in_directory(self, directory, file_type='*'):
```

Comment:
I think this makes more sense the way it was originally named, since we're not actually "getting" the bitstreams here, we're just creating a list of Bitstream items.